app/kumactl/cmd/config/testdata/config-control-planes-remove.13.golden.yaml

```diff
@@ -1,2 +1 @@
-contexts: []
-controlPlanes: []
+{}
```

Comment:
OK, that is interesting now.  Why is this?

Reply:
"Proto changes" in the readme

Reply:
Nevermind, I just read the PR description to the end.